### ISSUE TITLE
Easier tagging and adding tags already in the article overview. 

### ISSUE THREAD

**Issue by cukabeka** [wallabag/wallabag#5513]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I want to add tags to articles in the web frontend. I would like to have a feature to quickly add tags in the article overview.
I searched for other feature requests here in the issues and was surprised to find none regarding the tagging experience.

**Describe the solution you'd like**
I want to have an additional icon in the bottom row to tag an article already in the overview:
![image](https://user-images.githubusercontent.com/688274/147225061-2264a554-bff3-4985-b279-e3078287505e.png)

Additionally, in the article view, I'd expect a tagging possibility right at the top. I think the positioning in the flyout menu is not very elegant and takes a lot of klicks to add a tag.
![image](https://user-images.githubusercontent.com/688274/147225458-43a215a2-344e-4dd2-8ebf-6062be811058.png)

I'd like to be able to tag articles elegantly both on desktop and on touch devices.

**Describe alternatives you've considered**
The wallabagger chrome extension is providing this feature in an elegant way, including the autosuggestion of tags.

**Additional context**
I use tagging a lot to structure informations that I have collected with the articles. Tagging is the core feature for this, so I would appreciate a lot if the tagging experience would be simplified.

**Comment by gitmaker032** [comment]
I agree. It really surprises me that the only way to add a tag is to go into an article and select an item in the menu! It's long, not convenient. I'd really like to be able to add a tag immediately when adding an article, like entering it right after the address or selecting it in an additional quick menu. And of course there should be a button next to the other quick/bulk actions:

![image](https://user-images.githubusercontent.com/67278783/148352421-27e662da-0b89-4f30-9a7d-35608b937596.png)
![image](https://user-images.githubusercontent.com/67278783/148352753-12292d49-8da6-4c6d-9f6d-1b9c2d16a2fb.png)

I urge other users to support this request to add a feature. Seriously, the second version of the webpage saving app and there's no way to quickly add/remove a tag!? No way! 😀
